src/magnify/apps/core/models.py

```diff
@@ -343,6 +258,263 @@ def __str__(self):
         return f"{capfirst(self.room.name):s} / {capfirst(self.group.name):s}{admin_status:s}"
 
 
+class Meeting(BaseModel):
+    """Model for one meeting or a collection of meetings defined recursively"""
+
+    DAILY, WEEKLY, MONTHLY, YEARLY = "daily", "weekly", "monthly", "yearly"
+
+    INTERVAL_CHOICES = (
+        (DAILY, _("Daily")),
+        (WEEKLY, _("Weekly")),
+        (MONTHLY, _("Monthly")),
+        (YEARLY, _("Yearly")),
+    )
+
+    DATE_DAY, NTH_DAY = "date_day", "nth_day"
+    MONTHLY_TYPE_CHOICES = (
+        (DATE_DAY, _("Every month on this date")),
+        (NTH_DAY, _("Every month on the nth week day of the month")),
+    )
+
+    weekdays_validator = RegexValidator(
+        "^[0-6]{0,7}$",
+        message=_("Weekdays must contain the numbers of the active days."),
+    )
+
+    name = models.CharField(max_length=500)
+    room = models.ForeignKey(
+        Room, null=True, blank=True, related_name="meetings", on_delete=models.RESTRICT
+    )
+
+    # Start and end are the same for a single meeting
+    start = models.DateField()
+    start_time = models.TimeField()
+    expected_duration = models.DurationField()
+
+    # recurrence
+    recurrence = models.CharField(
+        max_length=10, choices=INTERVAL_CHOICES, null=True, blank=True
+    )
+    frequency = models.PositiveIntegerField(default=1)
+    until = models.DateField(null=True, blank=True)
```

Comment:
At first read, I didn't get what `until` means. Maybe use `recurring_until` ?

Reply:
I would like to keep it in one word. 
At first it was called `end` then I wanted to avoid confusion with the end of a meeting occurrence. 
Do you think `end` is better or do you have another idea in one word?

Reply:
@kernicPanel wdyt?

Reply:
No good reason... it's true that `recurring_until` is a good name. So `recurring_until` or `end`?

Reply:
`recurring_until` reflects better his purpose, I think.